skbio/io/format/genbank.py

```diff
@@ -429,7 +431,6 @@ def _parse_single_genbank(chunks):
         elif header == 'FEATURES':
             # merge the two dictionaries together. Note this is only
             # available in Python-3.5
-
             feature_metadata = merge_dicts(parsed, feature_metadata)
         else:
             metadata[header] = parsed
```

Comment:
how can that be? you just add an item to metadata. did you update the
expected result in the unit tests?

On Thu, Feb 25, 2016 at 1:38 PM, Jamie Morton <notifications@github.com>
wrote:

> In skbio/io/format/genbank.py
> <https://github.com/RNAer/scikit-bio/pull/5#discussion_r54169114>:
>
> > @@ -429,7 +431,6 @@ def _parse_single_genbank(chunks):
> >          elif header == 'FEATURES':
> >              # merge the two dictionaries together. Note this is only
> >              # available in Python-3.5
> > -
> >              feature_metadata = merge_dicts(parsed, feature_metadata)
> >          else:
> >              metadata[header] = parsed
>
> I think it is a better idea to leave this out.
>
> Its breaking a whole bunch of tests and I'm not entirely sure what this
> does.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RNAer/scikit-bio/pull/5/files#r54169114>.
>